TensorIO/Classes/TensorFlow/TIOTensorFlowData/TIOPixelBuffer+TIOTensorFlowData.mm

```diff
@@ -72,6 +91,102 @@ void TIOCopyCVPixelBufferToTensorFlowTensor(CVPixelBufferRef pixelBuffer, tensor
     CFRelease(pixelBuffer);
 }
 
+// TODO: ensure 16 byte pixel buffer alignment
```

Comment:
https://github.com/doc-ai/tensorio-ios/issues/1 longstanding but low priority until we start using inception models (299x299)